dotnet/src/SemanticKernel.Abstractions/Contents/AnnotationContent.cs

```diff
@@ -43,13 +42,17 @@ public AnnotationContent()
     /// <summary>
     /// Initializes a new instance of the <see cref="AnnotationContent"/> class.
     /// </summary>
+    /// <param name="quote">The source text being referenced.</param>
     /// <param name="modelId">The model ID used to generate the content.</param>
     /// <param name="innerContent">Inner content</param>
     /// <param name="metadata">Additional metadata</param>
     public AnnotationContent(
+        string quote,
```

Comment:
This parameter wasn't required previously, are there any changes that force to have it as required now? If it's not required, we can add it to the new constructor overload to avoid breaking changes.

Reply:
This class is produced by the assistant framework as output only.  I woudln't expect any caller to construct one.  Its marked as experimental also and we've taken some breaking changes for agents.

Reply:
I'm wondering if we should put this class in Agents assembly and make constructor internal in this case?

Reply:
For serialization, it has to be in abstractions.